core/controllers/payload_validator.py

```diff
@@ -27,15 +27,41 @@
 from typing import Any, Dict, List, Optional, Tuple, Union
 
 
+def get_schema_type(arg_schema: Dict[str, Any]) -> Any:
```

Comment:
```suggestion
def get_schema_type(arg_schema: Dict[str, Any]) -> str:
```

Also please add a comment explaining why the `arg_schema` is `Dict[str, Any]`

Reply:
Added.